fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rewrite/AddDecodeNodeForDictStringRule.java

```diff
@@ -413,8 +411,6 @@ public OptExpression visitPhysicalOlapScan(OptExpression optExpression, DecodeCo
                             scanOperator.getProjection());
                     newOlapScan.setPreAggregation(scanOperator.isPreAggregation());
                     newOlapScan.setGlobalDicts(globalDicts);
-                    newOlapScan.setGlobalDictStringColumns(globalDictStringColumns);
```

Comment:
please ensure this logical

Reply:
globalDictStringColumns is unused since 2.1.5/2.0.5